test/runtests.jl

```diff
@@ -341,7 +341,7 @@ end
 let weakdict = pyimport("weakref")["WeakValueDictionary"]
     # (use weakdict for the value, since Python supports
     #  weak references to type objects)
-    @test weakdict(Dict(3=>weakdict)) == Dict(3=>weakdict)
+    @test weakdict(Dict(3=>weakdict))[:__getitem__](3) == weakdict
```

Comment:
@malmaud, `getindex(o::PyObject, i::Integer)` is defined and calls Python's getitem method.